six/modules/c++/cphd/source/Wideband.cpp

```diff
@@ -230,6 +230,7 @@ void scale(const void* input,
 
 namespace cphd
 {
+
```

Comment:
Can anything in `cphd03/Wideband` be reused?

Reply:
I'm not sure how. Because the Wideband class stores a Data object. And this object is different for both CPHD03 and CPHD1.0, even though the functionality is identical for both, other than some compressed data handling in CPHD1.0.